frontend/packages/operator-lifecycle-manager/src/components/modals/uninstall-operator-modal.tsx

```diff
@@ -62,8 +90,52 @@ export const UninstallOperatorModal: React.FC<UninstallOperatorModalProps> = ({
   const removePlugins: boolean =
     !!consoleOperatorConfig && canPatchConsoleOperatorConfig && enabledPlugins.length > 0;
 
+  const subscriptionName = subscription?.spec?.name;
+  const subscriptionNamespace = subscription?.metadata?.namespace;
+
+  const isSubmitInProgress = operatorUninstallInProgress || operandsDeleteInProgress;
+
+  const isSubmitFinished =
+    !isSubmitInProgress &&
+    (deleteOperands ? operandsDeleteFinished : true) &&
+    operatorUninstallFinished;
+
+  const anySubmitErrors = React.useMemo(
+    () =>
+      anyOperandDeletionErrors(operandDeletionErrors) || operatorUninstallErrorMessage.length > 0,
+    [operandDeletionErrors, operatorUninstallErrorMessage],
+  );
```

Comment:
This is a simple calculation that returns a primitive. No need to memoize. It likely makes things slower here.

```suggestion
  const hasSubmitErrors = operandDeletionErrors.length > 0 || operatorUninstallErrorMessage.length > 0;
```